src/Output/ShellOutput.php

```diff
@@ -195,6 +195,8 @@ private function initFormatters()
 
         // Types
         $formatter->setStyle('number', new OutputFormatterStyle('magenta'));
+        $formatter->setStyle('integer', new OutputFormatterStyle('green'));
```

Comment:
Green is a super common color for output (because string) and magenta isn't used anywhere besides `number`. Let's keep it for one of the types (i'm thinking integer?) and use yellow for the other?

Reply:
Sure!